backend/src/create.rs

```diff
@@ -261,10 +261,7 @@ pub fn create_from_osm(
     };
     if let Some(mut demand) = demand {
         demand.finish_loading(&map.mercator);
-        map.impact = Some(Impact::new(&map, Some(&demand)));
```

Comment:
I haven't measured this, but it should slightly improve initial load time. Before, we picked random points per OD pair. Now we only do it when the user first goes to predict impact mode.